lambda-functions/notification-invoke-service/src/main/java/com/paladincloud/SendNotification.java

```diff
@@ -40,6 +40,26 @@ public String handleRequest(Map<String,Object> event, Context context)
                 System.out.println("bodyJson--"+bodyJson.toString());
                 payloadString = bodyJson.getAsJsonObject().get("Sns").getAsJsonObject().get("Message").getAsString();
             }
+
+            Object payloadObj = gson.fromJson(payloadString,Object.class);
+            if(payloadObj instanceof List){
```

Comment:
this line here is what allowing the single events to fall through here. This "if" statement is not working as intended for single objects